source/slang/slang-emit-hlsl.cpp

```diff
@@ -11,6 +11,23 @@
 namespace Slang
 {
 
+static const char* kHLSLBuiltInPrelude64BitCast = R"(
+uint64_t _slang_bitcast64(double x)
```

Comment:
Should be named to _slang_asuint64() to align with existing HLSL intrinsic

Reply:
ack, will update